mne/datasets/sleep_physionet/age.py

```diff
@@ -94,7 +104,28 @@ def fetch_data(subjects, recording=[1, 2], path=None, force_update=False,
     path = data_path(path=path, update_path=update_path)
     params = [path, force_update, base_url]
 
-    _check_subjects(subjects, 20)
+    _check_subjects(
+        subjects, 83, missing=[39, 68, 69, 78, 79], on_missing=on_missing)
+
+    # Check for missing recordings
```

Comment:
The following could go in a utils function if we keep the `on_missing` approach: